datasets/generated_reviews_enth/generated_reviews_enth.py

```diff
@@ -0,0 +1,100 @@
+"""`generated_reviews_enth`: Generated product reviews dataset for machine translation quality prediction, part of [scb-mt-en-th-2020](https://arxiv.org/pdf/2007.03541.pdf)"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@article{lowphansirikul2020scb,
+  title={scb-mt-en-th-2020: A Large English-Thai Parallel Corpus},
+  author={Lowphansirikul, Lalita and Polpanumas, Charin and Rutherford, Attapol T and Nutanong, Sarana},
+  journal={arXiv preprint arXiv:2007.03541},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+ `generated_reviews_enth`
+ Generated product reviews dataset for machine translation quality prediction, part of [scb-mt-en-th-2020](https://arxiv.org/pdf/2007.03541.pdf)
+ `generated_reviews_enth` is created as part of [scb-mt-en-th-2020](https://arxiv.org/pdf/2007.03541.pdf) for machine translation task.
+ This dataset (referred to as `generated_reviews_yn` in [scb-mt-en-th-2020](https://arxiv.org/pdf/2007.03541.pdf)) are English product reviews
+ generated by [CTRL](https://arxiv.org/abs/1909.05858), translated by Google Translate API and annotated as accepted or rejected (`correct`)
+ based on fluency and adequacy of the translation by human annotators.
+ This allows it to be used for English-to-Thai translation quality esitmation (binary label), machine translation, and sentiment analysis.
+"""
+
+
+class GeneratedReviewsEnthConfig(datasets.BuilderConfig):
+    """BuilderConfig for GeneratedReviewsEnth."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for GeneratedReviewsEnth.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(GeneratedReviewsEnthConfig, self).__init__(**kwargs)
+        self.language_pair = ("en", "th")
+
+
+class GeneratedReviewsEnth(datasets.GeneratorBasedBuilder):
+    """`generated_reviews_enth`: Generated product reviews dataset for machine translation quality prediction, part of [scb-mt-en-th-2020](https://arxiv.org/pdf/2007.03541.pdf)"""
+
+    _DOWNLOAD_URL = "https://github.com/vistec-AI/generated_reviews_enth/raw/main/data.zip"
+    _TRAIN_FILE = "train.jsonl"
+    _VAL_FILE = "valid.jsonl"
+    _TEST_FILE = "test.jsonl"
+
```

Comment:
```suggestion
    BUILDER_CONFIG_CLASS = GeneratedReviewsEnthConfig
```
we need this to specify that we need a custom config